starter/src/app/app.module.ts

```diff
@@ -2,16 +2,22 @@ import { NgModule, NgModuleFactoryLoader, SystemJsNgModuleLoader } from "@angula
 import { BrowserModule } from "@angular/platform-browser";
 import { FormsModule } from "@angular/forms";
 import { UIRouterModule } from "@uirouter/angular";
-import { StoreModule, MetaReducer, ActionReducerMap, ActionReducer } from "@ngrx/store";
+import { validateSync } from "class-validator";
+import { ActionReducer, ActionReducerMap, MetaReducer, StoreModule } from "@ngrx/store";
 import { storeFreeze } from "ngrx-store-freeze";
 import { storeLogger } from "ngrx-store-logger";
 import { BrowserAnimationsModule } from "@angular/platform-browser/animations";
+import { StarkValidationErrorsUtil } from "../../../packages/stark-core/src/util";
```

Comment:
There is a small issue in the import. Could you please change it for @nationalbankbelgium/stark-core please? 😊

Reply:
Damn! Ok, I've fixed it ;)